scripts/prepublish

```diff
@@ -18,6 +20,9 @@ done
 npm run clean
 npm run build
 
+# build reads version from package.json, which isn't updated until after this script is run by xyz
+sed -i 's#//  Ramda v'${PREVIOUS_VERSION}'#//  Ramda v'${VERSION}'#' -- dist/ramda.js
```

Comment:
On Darwin this needs to be `-i ''` rather than `-i`. Could we simply remove the version number from __dist/ramda.js__ to avoid this complexity?

Reply:
Hopefully no tooling depends on it, so I guess that means we could? Seems kind of nice though?

I could also just use the same bash replacement technique we use for the README.

Reply:
Updated to use bash replacement instead of sed. cc @buzzdecafe @CrossEye do you want to keep this line stating the version in `dist/ramda.js` or get rid of it?

Reply:
I'd rather keep it.

Reply:
> I'd rather keep it.

:+1: ok